addons/product/views/product_views.xml

```diff
@@ -26,7 +26,7 @@
                                 options='{"terminology": "archive"}'/>
                         </button>
                     </div>
-                    <field name="image_medium" widget="image" class="oe_avatar"/>
+                    <field name="image_original" widget="image" class="oe_avatar" options="{'preview_image': 'image_medium'}"/>
```

Comment:
image

Reply:
Allowing to write both on image and image_original as we decided is confusing... I obviously prefer image_original but then if we want to do `if 'image' in vals` on create or write to apply some logic, we also need to add or `'image_original' in vals`...

I think it might be better to set the write logic only on one of the two and stick to it everywhere.

Reply:
And the problem of using image, it the backend for the o2m, we won't have the original blob to compute the size of the file if we use image instead.

Reply:
Ok, avoid using image_original